gordo_components/data_provider/ncs_reader.py

```diff
@@ -39,11 +43,15 @@ def __init__(
         ----------
         threads : Optional[int]
             Number of threads to use. If None then use 1 thread
+        remove_status_codes: Optional[list]
+            Removes data with Status code(s) in the list. By default it removes data
+            with Status code 0. Can be overwritten from kwargs.
```

Comment:
Ditch the `Can be overwritten from kwargs.` From the perspective of this class it is overwritten by the explicit parameter your are documenting, not kwargs.